lib/action_controller/serialization/context.rb

```diff
@@ -0,0 +1,13 @@
+module ActionController
+  module Serialization
+    class Context
+      attr_reader :request_url, :query_parameters, :url_helpers
```

Comment:
hmm as far as I understand `default_url_options` needs to be defined on the same object where `url_helpers` are included in order to be taken in to account. Am I not understanding it correctly?